android/gradle.properties

```diff
@@ -21,6 +21,8 @@
 # $keytool -genkey -v -keystore ./status-keystore.jks -keyalg RSA -keysize 2048 -validity 10000 -alias status
 
 # Version requirements used throughout the Gradle scripts
+kotlinVersion=1.7.22
+kotlinToolsVersion=1.8.0
```

Comment:
I have to take a second look at this

Reply:
resolved here : https://github.com/status-im/status-mobile/pull/18563/commits/0c258508186f5ea5f23e3652d9bca6c452f3969e